mlfinlab/cross_validation/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+Functions derived from Chapter 7: Cross Validation
+"""
+
+from mlfinlab.cross_validation.cross_validation import \
```

Comment:
We are using PEP 328 going forward. So when we have a long chain of imports we use (). Example: 

`from mlfinlab.cross_validation.cross_validation import (ml_get_train_times, ml_cross_val_score, PurgedKFold)`